OpenSim/Simulation/Model/Appearance.h

```diff
@@ -37,22 +37,21 @@ class Model;
 //=============================================================================
 //=============================================================================
 /**
- * A class that holds the Display Attributes (Appearance) of an object displayed 
- * in the OpenSim Visualizer.
+ * A class that holds the Appearance Attributes of an object displayed 
+ * in the OpenSim Visualizer/GUI.
  * 
- * BaseAppearance contains Appearance properties that apply to all geometry.
+ * Appearance objects contain properties that are common to all geometry.
  * Geometry that have a surface so that it can be textured can use the subclass
```

Comment:
It is not clear from the class description why Appearance is needed rather than having properties (attributes) of Geometry. We need to state the difference between Geometry and Appearance. Geometry being a representation of physical shapes as a Component for computation and Appearance being a convenience class of knobs for Visualizers to use to control how that geometry is visualized if at all. Is that correct?